exquery-restxq-specification/restxq-1.0-specification.html

```diff
@@ -152,16 +152,16 @@
       <a href="http://exquery.org/" title="EXQuery: Extensions for EXQuery">EXQuery project</a>.
     </section>
     <section id='abstract'>
-      Whilst XQuery [[!XQUERY]] was originally envisaged and designed as a query language for XML,
+      Whilst XQuery was originally envisaged and designed as a query language for XML,
```

Comment:
I think this should be: `Whilst XQuery (see [[XQuery]]) was`

Reply:
This was causing validation issues as specified, possibly because the abstract/introduction counts as a normative section (see https://respec.org/docs/#normative-vs-informative-references); `[[?XQuery]]` seems to work instead?

Reply:
I did test this with `[[XQUERY]]` and it does fix the validation issue.

Reply:
But `[[XQUERY]]` shows up in the normative references section; is that what you want?  Otherwise `[[?XQUERY]]` populates the non-normative reference section.

Ditto for the other three references.  Let me know which (if any) you feel ought to be normative, otherwise I will leave them as non-normative. @adamretter

Reply:
It seems to me that XQuery is a dependency of RESTXQ, which would make XQuery a normative reference. Can you explain why you believe it is a non-normative dependency please @yamahito ?